### PR TITLE
Use the official SPDX licenses.json file

### LINKED ISSUE #2: Switch to use the official SPDX JSON data
By now there is https://spdx.org/licenses/licenses.json. The spdx-license-list project does not use it yet [because they need the full license text](https://github.com/sindresorhus/spdx-license-list/issues/6), but this project doesn't.

@domcleal So how about not depending on spdx-license-list and use https://spdx.org/licenses/licenses.json directly?

### PR DESCRIPTION
**Pull request by sschuberth**
This removes the dependency on the spdx-license-list project.

Fixes #2.

### CODE REVIEW

**Comment by andrew** [comment]
Looks great, 👍 for updating to v2.4 of the spdx license list too

File: lib/spdx-licenses.rb [hunk 1]
```diff
@@ -4,12 +4,24 @@
...
   def self.data
-    @@data ||= JSON.load(File.read(File.expand_path('../../spdx.json', __FILE__)))
+    unless defined?(@@data)
+      spdx = JSON.load(File.read(File.expand_path('../../licenses.json', __FILE__)))
+
+      # Convert the array of hashes to a hash with the license ID as the key and
+      # the rest of each hash as the value.
+      @@data = {}
+      spdx['licenses'].each do |details|
+          id = details.delete('licenseId')
```
Comment (domcleal) [inline lib/spdx-licenses.rb]: Nitpick, stick with 2-space indentation here.

**Comment by domcleal** [comment]
Great, just one inline comment and if you can rebase then we can check tests pass, as I've fixed it on master.

**Comment by sschuberth** [comment]
@domcleal I've addressed your comments, and thanks for fixing the tests!

**Comment by domcleal** [comment]
Merged, thanks @sschuberth!

I'll cut a new release shortly.

**Pull request merged**